docs/supported_ops.rst

```diff
@@ -1,14 +1,17 @@
 Supported operations
 ====================
+
+.. include:: op_inclusion
```

Comment:
I'm still against welcoming the user with this list and hiding any info below. 
Even if we were to change the order, I think there should be at least one or two sentences of introduction here.

Reply:
I think that the "Supported operations" title speaks for itself. This section lists supported operations... What kind of one-sentence-introduction you have in mind?
My proposal is based on my experience of actually using the docs.

Reply:
My point is you're trying to save one click in the docs list while making it harder to follow for anyone new to DALI.

Just scrolling through this page it won't be hard to miss the section you moved below the table (as table also links to operator docs directly).

Otherwise, I would maybe use something like:

> This page lists all the supported DALI operators, `backends/devices` _link to section about backends_ and `data kinds?` _link to all the Image, Audio, Sequence stuff_ that they can use/they support. You can read more on how to use operators in `Getting Started` that describes steps to define the `Pipeline` and its usage.

Alternatively maybe we should split all the additional info into its own dedicated subpage.

Still, I would just leave it as it is, so its hard for me to come up with a good intro.

> My proposal is based on my experience of actually using the docs.

You're not the only one :P